mautic/mautic
# Display just published fields in form field mapping

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Unpublished contact fields are displayed in form field mapping tab. This PR fixed it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Unpublish one of your contact fields, for example address 1
3. Go to form and try map this field with your form field
4. After PR is not possible




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10734"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
